scripts/sysman/driver.yml

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Tool APIs for System Resource Management (Sysman)"
+ordinal: "1"
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Supported sysman initialization flags"
+version: "1.5"
+class: $s
+name: $s_init_flags_t
+etors:
+    - name: PLACEHOLDER
+      desc: "placeholder for future use"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Initialize $OneApi System Resource Management (sysman)"
+version: "1.5"
+class: $s
+name: Init
+decl: static
+ordinal: "0"
+details: 
+    - "The application must call this function or $xInit with the $S_ENABLE_SYSMAN environment variable set before calling any other sysman function."
+    - "If this function is not called then all other sysman functions will return $X_RESULT_ERROR_UNINITIALIZED."
+    - "Only one instance of sysman will be initialized per process."
+    - "The application must call this function after forking new processes. Each forked process must call this function."
+    - "The application may call this function from simultaneous threads."
+    - "The implementation of this function must be thread-safe for scenarios where multiple libraries may initialize sysman simultaneously."
+params:
+    - type: $s_init_flags_t
+      name: flags
+      desc: |
+            [in] initialization flags.
+            currently unused, must be 0 (default).
+      init: "0"
+returns:
+    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Retrieves sysman driver instances"
+version: "1.5"
+class: $sDriver
+name: Get
+decl: static
+ordinal: "0"
+details:
+    - "A sysman driver represents a collection of physical devices."
+    - "Multiple calls to this function will return identical sysman driver handles, in the same order."
```

Comment:
@bashbaug : should we clarify behavior of the affinity mask with respect to this order?

Reply:
Yes, I think so.  IIRC we do NOT want the affinity mask to apply to sysman handles, correct?

Reply:
I added the line:

> * The number and order of handles returned from this function is NOT affected by the ZE_AFFINITY_MASK or ZE_ENABLE_PCI_ID_DEVICE_ORDER environment variables.

I can add back support for ZE_ENABLE_PCI_ID_DEVICE_ORDER if needed, but I'd prefer to have a clean separation if possible.